classes/app.rb

```diff
@@ -1,10 +1,12 @@
+require 'json'
 require_relative 'book'
 require_relative 'person'
 require_relative 'rental'
 require_relative 'student'
 require_relative 'teacher'
 require_relative 'classroom'
-
+require_relative 'preserve_data'
+# rubocop:disable Metrics/ClassLength
 class App
```

Comment:
- [ ] It would be nice if you don't disable the linters because linters prevents your from making those errors in your app which can break your app completely 🤔 Kindly remove this disable rubocop and try to fix the errors if there is any 👍